_episodes/01-introduction.md

```diff
@@ -22,40 +21,28 @@ keypoints:
 This section outlines the structure of this two-day course and provides some
 essential terms for discussion.
 
-## A Master Class
+## Practice and Feedback
 
-Many metaphors can be used to describe teaching and learning: filling
-up a container, lighting a fire, growing a seed, and more.  Our
-favorite is to compare teachers to actors, musicians, athletes, and
-other performing artists.  To become an accomplished performer, one
```

Comment:
that sentence was definitely me going "I need to start this introduction...but how?"